### ISSUE TITLE
Headings on "Send your documents" in comfirmation-further actions

### ISSUE THREAD

**Issue by kitation** [UKHomeOffice/design-system#167]
This component uses a h3 for the filename but this should be a h2 as it sits directly under the H1 of the page. 
<img width="445" alt="Screenshot 2020-07-22 at 15 34 32" src="https://user-images.githubusercontent.com/1537694/88189427-f3436c00-cc30-11ea-99b4-c4641670b371.png">

**Comment by joeedhorton** [comment]
Good spot, I have included it in our long list of accessibility issues that will be attacked soon! 👍 

See issue [here](https://github.com/UKHomeOffice/design-system/issues/168)
